app/views/visitor_mailer/request_acknowledged.html.erb

```diff
@@ -0,0 +1,67 @@
+<p>
+Dear <%= @visit.visitor_first_name %>,
+</p>
+<h1>
+  Your visit is not booked yet
+</h1>
+<p>
+You can check whether your visit has been booked on the <%= link_to('TODO', '#') %>
+</p>
+<p>
+We'll email you by TODO to confirm the date and time of your visit. Please don't contact us any sooner than that, as we won't be able to comment on booking requests that are being processed.
+</p>
+<p>
+If you don’t receive a confirmation email by TODO, please check your spam or junk folder for it. You need to do this on a computer or tablet (rather than a smart phone).
+</p>
+<h1>
+  Add this email to your address book
+</h1>
+<p>
+To stop visit confirmation emails going to your spam or junk folder please add no-reply@email.prisonvisits.service.gov.uk to your address book or safe senders list.
+</p>
+<h1>
+  Visit request details
+</h1>
+<p>
+You have requested a visit to <%= @visit.prison_name %> for <%= pluralize(@visit.total_number_of_visitors, 'person', 'people') %>
+</p>
+<p>
+<% @visit.slots.each_with_index do |slot, index| %>
+  <% unless index == 0 %>
+    <br/>
+  <% end %>
+  <% if index == 0 %>
+    First choice:
```

Comment:
For reference, I've punted on the First/Alternative distinction and made them all 'Choice 1/2/3' - I think we'll eventually want to improve that, but I wanted to avoid a whole load of ad-hoc implementations